### PR TITLE
64 release process

### PR DESCRIPTION
**Pull request by IagoLast**
#64 
---

This adds a new script:

    npm run release

this script has all we need to publish new airship versions

### CODE REVIEW

File: packages/components/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@carto/airship-components",
-  "version": "0.0.0",
+  "version": "1.0.0-alpha.26",
```
Comment (VictorVelarde) [inline packages/components/package.json]: Reading lerna docs, it seems that we could skip individual version for the packages, as we would follow **fixed mode** (both packages always coordinated): https://www.npmjs.com/package/lerna#fixedlocked-mode-default Right?

Comment (IagoLast) [reply packages/components/package.json]: I think fixed mode makes all packages to have the same version

Comment (VictorVelarde) [reply packages/components/package.json]: Correct, didn't we want just that @jesusbotella ?

Comment (IagoLast) [reply packages/components/package.json]: yes :)

Comment (jesusbotella) [reply packages/components/package.json]: Yep! 😉

**Comment by jesusbotella** [comment]
I am kinda spoiled by prettifier or whatever it is, because I have seen that we changed the properties' order in `package.json` in several PRs. 😞

**Comment by IagoLast** [comment]
so can we 🚀  this?

**Comment by IagoLast** [comment]
@VictorVelarde what do you mean? the version is auto-inyected by lerna

**Pull request merged**